samples/nodejs_simple_sample/src/gateway_sample_win.json

```diff
@@ -14,7 +14,7 @@
             "loader": {
                 "name": "node",
                 "entrypoint": {
-                    "main.path": "../../../samples/nodejs_simple_sample/nodejs_modules/printer.js"
```

Comment:
This is the difference between running it from the command line and running it from VS.